e2store/src/era1.rs

```diff
@@ -118,6 +118,10 @@ impl Era1 {
         file.write(&mut buf)?;
         Ok(buf)
     }
+
+    pub fn epoch_number(&self) -> u64 {
+        self.block_index.block_index.starting_number / 8192
```

Comment:
I will do 1 and 2, but leave `self.block_tuples[block_number % EPOCH_SIZE]` for now

Reply:
I don't think these make sense in Era1, eventually we will handle era files as well, but we can do this for now then see what happens when we get there

Reply:
I think it makes sense because that's the place that **should** know how many blocks are in each era1 file, and in which file and at what index the specific block is. Other crates shouldn't have to implement that calculation (even tho it's simple).

Maybe not that specific file, but definitely somewhere in that crate.